src/Standards/Squiz/Sniffs/WhiteSpace/LanguageConstructSpacingSniff.php

```diff
@@ -50,17 +51,21 @@ public function process(File $phpcsFile, $stackPtr)
     {
         $tokens = $phpcsFile->getTokens();
 
+        if (isset($tokens[($stackPtr + 1)]) === false) {
+            // Skip if there is no next token.
+            return;
+        }
+
         if ($tokens[($stackPtr + 1)]['code'] === T_SEMICOLON) {
             // No content for this language construct.
             return;
         }
 
         if ($tokens[($stackPtr + 1)]['code'] === T_WHITESPACE) {
-            $content       = $tokens[($stackPtr + 1)]['content'];
-            $contentLength = strlen($content);
-            if ($contentLength !== 1) {
-                $error = 'Language constructs must be followed by a single space; expected 1 space but found %s';
-                $data  = array($contentLength);
+            $content = $tokens[($stackPtr + 1)]['content'];
+            if ($content !== ' ') {
+                $error = 'Language constructs must be followed by a single space; expected 1 space but found "%s"';
+                $data  = array($tokenContent = Util\Common::prepareForOutput($content));
```

Comment:
Personally I avoid assignments that can be tricky to the eye. I would've added the assignment on a different line.